src/sgame/botlib/bot_nav.cpp

```diff
@@ -112,11 +112,34 @@ void G_BotEnableArea( const glm::vec3 &origin, const glm::vec3 &mins, const glm:
 	BotSetPolyFlags( &origin[0], &mins[0], &maxs[0], POLYFLAGS_WALK );
 }
 
-void G_BotSetNavMesh( int botClientNum, qhandle_t nav )
+void G_BotSetNavMesh( int botClientNum, class_t newClass )
 {
-	if ( nav < 0 || nav >= numNavData )
+	if ( newClass == PCL_NONE )
 	{
-		Log::Warn( "Navigation handle out of bounds" );
+		return;
```

Comment:
warn?

Reply:
Warn what? The page only displays 1 context line and I can't figure out how to see the code this comment is attached to...

Reply:
You refactored the code so that this comment no longer applies. I meant don't silently return when no navmesh is found.